src/components/appointment/AppointmentHostCard.tsx

```diff
@@ -0,0 +1,98 @@
+import { AppointmentDetail } from "@/types/Appointment";
+import { CardContent, Grid, Stack, Typography, Box, Chip } from "@mui/material";
+import Image from "next/image";
+import BorderWithShadow from "../public/BorderWithShadow";
+import AppointmentHeader from "./AppointmentHeader";
+
+const chipsStyle = {
+  minWidth: 100,
+  height: 40,
+  fontSize: 18,
+  fontWeight: 'bold',
+  border: "black solid 4px",
+  borderRadius: "16px",
+  boxShadow: "4px 4px #BFBFBF",
+}
+type props = {
+  appointmentDetail: AppointmentDetail,
+};
+export default function AppointmentHostCard(props: props) {
+  return <Grid container
+    spacing="40px"
+    width="80vw"
+  >
+    <Grid item xs={12} md={6} style={{ display: "flex" }}>
+      <BorderWithShadow>
+        <CardContent>
+          <Stack spacing="12px" sx={{ padding: "20px", height: "100%" }}>
+            <AppointmentHeader detailHeader={props.appointmentDetail.detailHeader} isHost={true} />
+            <Box >
+              <Typography variant="h5" sx={{ fontWeight: "bold", marginBottom: "4px" }}>Image</Typography>
```

Comment:
I think if there's no image in the appointment, The `Image` header should invisible.
<img width="483" alt="image" src="https://user-images.githubusercontent.com/62459518/222941288-c8465d51-af76-441a-ab3f-41125f73304f.png">

Reply:
done solve